packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts

```diff
@@ -87,8 +87,13 @@ export class EditorSettingsSidebarComponent {
 	 * Sets the post visibility to the provided visibility setting.
 	 *
 	 * @param {PrivacyOptions} visibility Desired post visibility setting.
+	 * @param param1 Object parameter.
+	 * @param {string} param1.password Password for the post.
```

Comment:
We should call out here that this is required for Password option (but not the others). It may seem pretty obvious, but I still think that kind of faux-requirement is worth calling out in comment headers!

Reply:
Good point, thanks!

Reply:
A part of me wonders if there's an easier way to enforce this kind of conditional dependency at the parameter level. To your knowledge, does TypeScript offer something like that?